packages/styled-components/src/models/StyledNativeComponent.js

```diff
@@ -44,15 +45,20 @@ class StyledNativeComponent extends Component<*, *> {
             this.props
           );
 
+          const isTargetTag = isTag(elementToBeRendered);
+          const computedProps = this.attrs !== props ? { ...props, ...this.attrs } : props;
+          const propFilterFn = shouldForwardProp || (isTargetTag && validAttr);
```

Comment:
So for native since there's no "default" prop whitelist to speak of, the default validator fn can just be a function that returns `true`.

Reply:
Does https://github.com/styled-components/styled-components/pull/3107/commits/a3569b2ca0a2e2cd7de2a1104662d60a7a45d29c address this? Updated the comments slightly.